src/bindings/html/view.js

```diff
@@ -85,28 +79,42 @@ function onMutations(mutations) {
 }
 
 export function translateDocument(view, langs) {
-  const doc = view._doc;
+  const html = view._doc.documentElement;
 
-  if (langs[0].code === doc.documentElement.getAttribute('lang')) {
-    return Promise.resolve().then(
-      () => dispatchEvent(doc, 'DOMLocalized', langs));
+  if (readiness.get(html)) {
```

Comment:
With the current code, `get` and `has` are equivalent.  I don't expect us to set anything else than `true`, so I can go with `has`.